cfme/tests/test_soap.py

```diff
@@ -460,6 +460,10 @@ def test_vm_tagging(self, soap_client):
 
 
 class TestProvisioning(object):
+    WAIT_TIME = 300                         # For the VM status checking functions
+    WAIT_TIME_SLOW = 600                    # For slow providers
+    SLOW_PROVIDERS = {"rhevm", "scvmm"}     # That is what I was speaking about
```

Comment:
Kill the inline comments, please. Also, I have no idea what you were speaking about.